torch/nn/modules/dropout.py

```diff
@@ -234,7 +243,9 @@ def forward(self, input: Tensor) -> Tensor:
 
 
 class FeatureAlphaDropout(_DropoutNd):
-    r"""Randomly masks out entire channels (a channel is a feature map,
+    r"""Randomly masks out entire channels.
+
+    (a channel is a feature map,
     e.g. the :math:`j`-th channel of the :math:`i`-th sample in the batch input
     is a tensor :math:`\text{input}[i, j]`) of the input tensor). Instead of
```

Comment:
```suggestion
    A channel is a feature map,
    e.g. the :math:`j`-th channel of the :math:`i`-th sample in the batch input
    is a tensor :math:`\text{input}[i, j]` of the input tensor). Instead of
```